### DISCUSSION TITLE
missing npm packages while installing puppeteer@21.5.2 with cft 119.0.6045.105

### DISCUSSION THREAD

**Discussion by Karthikeyan2752** [nodejs/help#4671] [Q&A]
am trying to update the puppeteer from 20.9.0 to 21.5.2 for and chrome for testing to 119_0_6045_105, for the puppeteer installatio i am using this command and used node js 16.16.0 before and upgrading to 20.10.0 "$node_bin_path"/npm install puppeteer@"$puppeteer_version" --omit=dev --prefix "$node_modules_path" if remove prefix command, it is working fine, but i want the install the puppeteer in a specific directory, help me sort out the issuue, INSTALLATION LOG: `

     [exec] npm verb statusCode 404
     [exec] npm verb pkgid @npmcli/docs@^1.0.0
     [exec] npm verb cwd /home/puppeteer/pdf/build
     [exec] npm verb Linux 6.2.0-39-generic
     [exec] npm verb node v20.10.0
     [exec] npm verb npm  v10.2.3
     [exec] npm ERR! code E404
     [exec] npm ERR! 404 Not Found - GET <npm regsitry url>/@npmcli%2fdocs - no such package available
     [exec] npm ERR! 404 
     [exec] npm ERR! 404  '@npmcli/docs@^1.0.0' is not in this registry
     
i have tried insatalling other versions of node, the samed error is showing, and i tried with 120 stable chrome for testing version it is working fine, but the puppeteer suggest only 119 chrome version, when i install the puppeteer globally it is working fine, but showing error when we try to install it in a specific directory using --prefix. help me.

**Comment by marco-ippolito** [comment]
try to `npm set registry https://registry.npmjs.org/`, also it would be great to see the debug.log

**Comment by Karthikeyan2752** [comment]
On Thu, 11 Jan 2024 at 4:31 PM, Marco Ippolito ***@***.***>
wrote:

> try to npm set registry https://registry.npmjs.org/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/nodejs/discussions/51436#discussioncomment-8093350>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A272HD6VHAU6JKSNRJVMLQDYN7A73AVCNFSM6AAAAABBWBRUAOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAOJTGM2TA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
Tried, same packages missing error is coming, but it is working fine with
120 cft version. I am installing all these using - - prefix command.
